2025-09-04 changes

0xldr opened the pull request:
Adds changes for the 2025-09-04 executive as well as adding a missing Morpho doc, and correcting an incorrect title for the susds/usdt pool on Curve.

Missing Aave docs are noted and should be added this week.

boet1 commented:
Looks good to me

0xldr committed:
add previously missed Spark USDS Morpho Vault

usds

0xldr committed:
fix incorrect title, curve pool uses susds, not susdc

0xldr committed:
add changes for 2025-09-04 executive

boet1 commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -30361,6 +30539,298 @@ <h1>Agent Scope Database</h1>
+          </td>
+          <td>Spark</td>
+          <td>Core</td>
+          <td><code>0xe41a0583334f0dc4E023Acd0bFef3667F6FE0597</code></td>
```
The only observation I found: Is it correct to use the same address as in Morpho USDS?

0xldr replied:
Good catch, the vault is actually being created in the spell so we don't have the address, we'll make this a placeholder for now

0xldr replied:
fixed in https://github.com/sky-ecosystem/next-gen-atlas/pull/55/commits/35c06baa5ac34b09cadd82c5f6b9cfb949a42f10

0xldr committed:
add placeholder for new morpho vault address

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -28113,6 +28137,160 @@ <h1>Agent Scope Database</h1>
         </tr>
+        <tr>
+          <td>
+            <dfn>Ethereum Mainnet - Aave USDe Instance Configuration Document</dfn>
```
```suggestion
            <dfn>Ethereum Mainnet - Aave aUSDe Instance Configuration Document</dfn>
```

I think this should be aUSDe, if that is correct then there are a couple of instances that must be updated.

0xldr replied:
It was written this way to remain consistent with the USDC instances. Either is fine, but we should update all of them.

We would lean toward using "USDe" instead of "aUSDe" as it makes it references directly the asset being deposited. However, if AA want us to switch them all to `aUSDe` we can.

0xldr replied:
Logically it would follow that we would also need to update all Curve, SparkLend and Morpho instance titles to reference the receipt token instead of the deposit token.

aa-lex replied:
Okay, no keep it as you have it then and we can do an edit for all of the relevant protocols later if needed.

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -27491,8 +27515,8 @@ <h1>Agent Scope Database</h1>
             <ul>
-              <li><code>maxAmount</code>: 50,000,000</li>
-              <li><code>slope</code>: 25,000,000</li>
+              <li><code>maxAmount</code>: 100,000,000</li>
```
```suggestion
              <li><code>maxAmount</code>: 100,000,000 USDT</li>
```

0xldr replied:
Units were not included to remain consistent with other instances. Do you want us to update all of them?

aa-lex replied:
I added in the units that were specified in the Executive Document. I think we should aim for getting the units included, but we could also update the units later in an edit if you think it is too much of a discrepancy.

0xldr replied:
done

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -27491,8 +27515,8 @@ <h1>Agent Scope Database</h1>
-              <li><code>maxAmount</code>: 50,000,000</li>
-              <li><code>slope</code>: 25,000,000</li>
+              <li><code>maxAmount</code>: 100,000,000</li>
+              <li><code>slope</code>: 100,000,000 per day</li>
```
```suggestion
              <li><code>slope</code>: 100,000,000 USDT per day</li>
```

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -27636,8 +27660,8 @@ <h1>Agent Scope Database</h1>
             <ul>
-              <li><code>maxAmount</code>: 50,000,000</li>
-              <li><code>slope</code>: 25,000,000</li>
+              <li><code>maxAmount</code>: 100,000,000</li>
```
```suggestion
              <li><code>maxAmount</code>: 100,000,000 pyUSD</li>
```

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -27636,8 +27660,8 @@ <h1>Agent Scope Database</h1>
-              <li><code>maxAmount</code>: 50,000,000</li>
-              <li><code>slope</code>: 25,000,000</li>
+              <li><code>maxAmount</code>: 100,000,000</li>
+              <li><code>slope</code>: 100,000,000 per day</li>
```
```suggestion
              <li><code>slope</code>: 100,000,000 pyUSD per day</li>
```

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -30361,6 +30539,298 @@ <h1>Agent Scope Database</h1>
+          <td>Core</td>
+          <td>The inflow rate limits are:
+            <ul>
+              <li><code>maxAmount</code>: 50,000,000</li>
```
```suggestion
              <li><code>maxAmount</code>: 50,000,000 USDC</li>
```

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -30361,6 +30539,298 @@ <h1>Agent Scope Database</h1>
+          <td>The inflow rate limits are:
+            <ul>
+              <li><code>maxAmount</code>: 50,000,000</li>
+              <li><code>slope</code>: 25,000,000 per day</li>
```
```suggestion
              <li><code>slope</code>: 25,000,000 USDC per day</li>
```

aa-lex reviewed (commented):
Left a couple of suggestions :)

0xldr committed:
fix target protocol

0xldr committed:
Add missing Aave Core USDT instance

added here: https://vote.makerdao.com/executive/template-executive-vote-star2-allocation-system-updates-increase-gsm-pause-delay-add-emergency-spell-to-chainlog-top-up-of-the-integration-boost-launch-project-funding-spark-proxy-spell-star2-proxy-spell-april-30-2025

0xldr committed:
susdc > susds

0xldr committed:
add sUSDS as underlying asset of susdsusdt pool

0xldr committed:
fix susdsusdt parameters

0xldr committed:
add missing Curve USDC/USDT instance

added here: https://vote.makerdao.com/executive/template-executive-vote-sp-beam-initialization-sky-token-rewards-rebalance-set-aave-lido-market-prime-market-ddm-dc-to-0-sbe-changes-launch-project-funding-spark-aave-revenue-share-payment-ad-compensation-atlas-core-development-payments-spark-proxy-spell-april-17-2025

0xldr committed:
Add missing Aave Core USDS and USDC markets

added in: https://vote.makerdao.com/executive/template-executive-vote-wbtc-oracle-migration-allocator-spark-a-maximum-debt-ceiling-increase-integration-boost-funding-bug-bounty-payments-resilience-research-project-funding-aligned-delegate-compensation-atlas-core-development-funding-spark-proxy-spell-january-9-2025

0xldr committed:
add units for SLL instances

0xldr committed:
checksum all addresses

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -24068,12 +24100,21 @@ <h1>Agent Scope Database</h1>
+        </tr>
+        <tr>
+          <td>
+            <dfn>Ethereum Mainnet - Curve USDCT/USDT Pool Instance Configuration Document Location</dfn>
```
```suggestion
            <dfn>Ethereum Mainnet - Curve USDC/USDT Pool Instance Configuration Document Location</dfn>
```

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -27062,8 +27119,8 @@ <h1>Agent Scope Database</h1>
-              <li><code>maxAmount</code>: 200,000,000</li>
-              <li><code>slope</code>(per day): 400,000,000</li>
+              <li><code>maxAmount</code>: 200,000,000 USDS</li>
+              <li><code>slope</code>(per day): 400,000,000 USDS per day</li>
```
```suggestion
              <li><code>slope</code>: 400,000,000 USDS per day</li>
```

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -28277,35 +28326,27 @@ <h1>Agent Scope Database</h1>
           <td>Spark</td>
           <td>Core</td>
-          <td>The documents herein define the parameters of the Ethena USDe Instance of the Allocation System Primitive.</td>
+          <td>The documents herein define the parameters of the Aave Core USDT Instance of the Allocation System Primitive.</td>
```
```suggestion
          <td>The documents herein define the parameters of the Aave Core USDS Instance of the Allocation System Primitive.</td>
```

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -29781,23 +30454,353 @@ <h1>Agent Scope Database</h1>
           <td>Spark</td>
           <td>Core</td>
-          <td>USTB</td>
+          <td>sUSDSUSDT</td>
```
Shouldn’t this be USDCUSDT?

0xldr replied:
that's the market, Curve refer to it as crv2pool so we'll change to that

aa-lex reviewed (commented):
Left some small comments :)

0xldr committed:
Add Spark Blue Chip Vault address

https://etherscan.io/tx/0x0032e26b8e4b284e3c61ea8aeb0870e3f0dbb7d3173945faf0449ca6ec5138e8#eventlog#220

0xldr committed:
Lex review comments

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -28199,23 +28248,23 @@ <h1>Agent Scope Database</h1>
         </tr>
         <tr>
           <td>
             <dfn>Pool</dfn>
```
I don’t think there is a pool for this instance? If not, this document should be removed

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -28361,23 +28402,23 @@ <h1>Agent Scope Database</h1>
         <tr>
           <td>
-            <dfn>EthenaMinter</dfn>
+            <dfn>Pool</dfn>
```
I don’t think there is a pool for this instance? If not, this document should be removed

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -28435,235 +28476,185 @@ <h1>Agent Scope Database</h1>
         <tr>
           <td>
-            <dfn>Encode Function</dfn>
+            <dfn>Pool</dfn>
```
I don’t think there is a pool for this instance? If not, this document should be removed

aa-lex commented on Sky Atlas/Sky Atlas.html:
```diff
@@ -28719,23 +28710,23 @@ <h1>Agent Scope Database</h1>
         <tr>
           <td>
-            <dfn>EthenaMinter</dfn>
+            <dfn>Pool</dfn>
```
I don’t think there is a pool for this instance? If not, this document should be removed

0xldr committed:
remove pool documents that aren't needed

0xldr committed:
resolve conflicts

aa-lex reviewed (dismissed):
Looks great

0xldr committed:
checksum

aa-lex reviewed (approved):
Looks great @0xldr
